src/60_Samples_%26_Templates/Hotel.html

```diff
@@ -17,279 +18,149 @@
-        </div>
-        <div>
-            <amp-img src="/img/hotel02_joepitha_640x383.jpg" layout="fill" alt="Buckingham Palace gate" attribution="Joe Pitha on Flickr - CC BY-SA 2.0"></amp-img>
-            <div class="caption">Photo by <a href="https://flic.kr/p/ew3kma">Joe Pitha</a> / <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC BY-SA 2.0</a></div>
-        </div>
-        <div>
-            <amp-img src="/img/hotel03_muffinn_640x383.jpg" layout="fill" alt="The Victoria Memorial" attribution="muffinn on Flickr - CC BY 2.0"></amp-img>
-            <div class="caption">Photo by <a href="https://flic.kr/p/cuDKc3">muffinn</a> / <a href="https://creativecommons.org/licenses/by/2.0/">CC BY 2.0</a></div>
-        </div>
-        <div>
-            <amp-img src="/img/hotel04_robinhawkes_640x383.jpg" layout="fill" alt="Aerial view of Buckingham Palace" attribution="Robin Hawkes on Flickr - CC BY-SA 2.0"></amp-img>
-            <div class="caption">Photo by <a href="https://flic.kr/p/bw8B7r">Robin Hawkes</a> / <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC BY-SA 2.0</a></div>
-        </div>
-    </amp-carousel>
-    <p>Located next to London's Belgrave district Buckingham Palace offers a unique opportunity to stay in a piece of England's history. Traditionally the home of the UK's ruling sovereign since 1837 a small number of rooms are now available for rent. Spacious
-        high-ceilinged rooms offer four poster beds and views across St James Park. WiFi is limited.</p>
-</div>
-<!-- ## User status -->
+<script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
 <!--
-  The `amp-access` element checks the user's current session. The JSON response includes the user's loyalty status and how many free nights they have earned.
-  ```json
-  {
-    "username":"test-user",
-    "status":"Gold",
-    "freenights":2
-  }
-  ```
+  We use `amp-list` to show a list of items.
 -->
-<div>
-    <section amp-access="status = 'Gold'">
-        <template amp-access-template type="amp-mustache">
-            <div class="loyalty-message">Congratulations <strong>{{username}}</strong>, your PAMPered <strong>{{status}}</strong> status has earned you <strong>{{freenights}}</strong> free nights you can apply to this stay.</div>
-        </template>
-    </section>
-</div>
-<!-- ## Dates of stay -->
+<script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
 <!--
-  An amp-form is used to allow the user to change the dates of their stay. The `type="date"` is used to allow the browser to display a native date picker. Learn more about handling the result of a form submission in the [form example](/components/amp-form).
+  We use `amp-bind` to bind the value of the room list to a variable that can change dinamically.
```

Comment:
suggestion: ' We use `amp-bind` to filter rooms based on their availability.

Reply:
done